taqueria-protocol/package.json

```diff
@@ -5,7 +5,7 @@
 	"main": "index.js",
 	"scripts": {
 		"test": "echo \"Error: no test specified\" && exit 1",
-		"build": "npx tsc -noEmit -p ./tsconfig.json && npx tsup"
+		"build": "npx tsc --emitDeclarationOnly -p ./tsconfig.json && npx tsup"
```

Comment:
Not sure if we need this flag since it's already set to true in the tsconfig.json file 🤔

Reply:
Probably not. Any thoughts on where to provide this instruction - in the build script, or in the tsconfig file?

Reply:
If we are using `-p ./tsconfig.json` I feel like we should put it there. But I am good either way though